RHDEVDOCS-6267 (#92126)
openshift/openshift-docs: configuring/configuring-build-runs.adoc

toc::[]

> In a `BuildRun` custom resource (CR), you can define the build reference, build specification,

Reviewer: ```suggestion
In a `BuildRun` custom resource (CR), you can configure a build run by defining the build reference, build specification, parameter values, service account, output, retention parameters, and volumes to configure a build run. You can also configure build pods by using a `Build` CR.
```
From the [OCP doc guidelines](https://92126--ocpdocs-pr.netlify.app/openshift-builds/latest/configuring/configuring-build-runs.html
https://92126--ocpdocs-pr.netlify.app/openshift-builds/latest/configuring/configuring-openshift-builds.html): "Do not hard wrap lines at 80 characters (or at any other length)."

> parameter values, service account, output, retention parameters, and volumes to configure a build run.

Reviewer: ```suggestion

```
If you implement my first suggestion, you can remove this line.

> You can also use `Build` CR to configure build pods.

Reviewer: ```suggestion

```
If you implement my first suggestion, you can remove this line.
